tests/pytype_test.py

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env python
-"""Test runner for typeshed.
+r"""Test runner for typeshed.
```

Comment:
"r"? Like, in a regexp?

Reply:
r for raw string, since it has a literal backslash within it